backend/routes.js

```diff
@@ -715,6 +788,7 @@ route.POST('/kv/{contractID}/{key}', {
   const existingSize = existing ? Buffer.from(existing).byteLength : 0
   await sbp('chelonia.db/set', `_private_kv_${contractID}_${key}`, request.payload)
   await sbp('backend/server/updateSize', contractID, request.payload.byteLength - existingSize)
+  await appendToIndexFactory(`_private_kvIdx_${contractID}`)(key)
```

Comment:
Seems a bit wasteful to create a factory function on every call to updating a key.. I thought the point of a factory function was to save it for later use on-demand? Can this be cached somewhere so that we don't re-create it on each call?

Reply:
I agree in principle, but this is the best approach I could find. The reason they can't be cached is that they have a dynamic component (the `contractID`)

Reply:
Maybe I can export a dynamic and a factory version of those functions?

Reply:
nah it's fine